ynr/apps/elections/uk/templates/candidates/person-view.html

```diff
@@ -142,7 +142,13 @@ <h2>Candidacies:</h2>
         {% else %}
            Contesting the {{ election_data.name }} ({{ election_data.election_date }})
         {% endif %}</dt>
+       
       <dd>{{ person|post_in_election:election_data }}</dd>
+      {% for membership in person.memberships.all %}
+          {% if membership.deselected %}
+            <dd>This candidate has been deselected by their party, but will remain on the ballot paper. <a href="{{ membership.deselected_source }}">Learn more<a>.</dd>
```

Comment:
I don't think `deselected_source` is in the admin interface, and it's also not required. Can you:

1. Add it to the admin interface
2. wrap the link in a conditional so we don't show it if there's no source?